src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -204,32 +310,34 @@ protected function formatChannels(array $channels)
     }
 
     /**
-     * Get the public token value from the Ably key.
+     * Build an Ably message object for broadcasting.
      *
-     * @return mixed
+     * @param $event string
+     * @param $payload array
+     * @return \Ably\Models\Message
      */
-    protected function getPublicToken()
+    protected function buildAblyMessage($event, $payload = [])
     {
-        return Str::before($this->ably->options->key, ':');
+        return tap(new AblyMessage, function ($message) use ($event, $payload) {
+            $message->name = $event;
+            $message->data = $payload;
+            $message->connectionKey = data_get($payload, 'socket');
+        });
     }
 
     /**
-     * Get the private token value from the Ably key.
-     *
-     * @return mixed
+     * @param $channelName string
+     * @param $connectionId string
+     * @param $userId string
```

Comment:
```suggestion
     * @param  string  $channelName
     * @param  string  $connectionId
     * @param  string|null  $userId
```
First variable type and then the variable name
Mark `$userId` as nullable

Reply:
Fixed!